app/bundles/EmailBundle/EventListener/EmailSubscriber.php

```diff
@@ -24,29 +24,14 @@
 
 class EmailSubscriber implements EventSubscriberInterface
 {
-    /**
-     * @var AuditLogModel
```

Comment:
Don't get me wrong. I hate useless comments. But these property annotations are valuable for some IDEs. Could you please add them back?

Reply:
Sure, sorry for that. I think just my IDE doesn't like them. Reverted.

Reply:
Thanks!